addons/event_product/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+{
+    'name': 'Events Product',
+    'version': '1.3',
```

Comment:
Let's start afresh !
```suggestion
    'version': '1.0',
```